---
title: University of Idaho SARS-CoV-2
authors: "Kimberly R. Andrews, Daniel D. New, Digpal S. Gour, Kane Francetich,  Scott A. Minnich, Barrie D. Robison, Carolyn J. Hovde"
authorLinks: ""
affiliations: "Correspondence: kimberlya@uidaho.edu; University of Idaho Institute for Interdisciplinary Data Sciences (IIDS) Genomics and Bioinformatics Resources Core (GBRC)"
date: "September 2022"
dataset: "http://localhost:4000/UofISARSCoV2_UofI-2020-2021-AcademicYear?d=tree&f_population=Community,University&legend=open"
abstract: "This is an interactive narrative for a study using viral genome sequences to investigate SARS-CoV-2 transmission dynamics for the University of Idaho (students, staff, faculty) and the surrounding community during the 2020-2021 academic year. Scroll down to learn more."
---

# [Phylogenetic tree](http://localhost:4000/UofISARSCoV2_UofI-2020-2021-AcademicYear?d=tree&f_population=Community,University&legend=open)
This phylogenetic tree was created using SARS-CoV-2 genome sequences for samples collected from the University of Idaho, the surrounding community, and other locations around the world. The tips of the tree represent the genome sequences (green dots for "University," blue dots for "Community," and no dots for other locations). The lengths of the horizontal branches indicate the level of genetic difference between sequences, and the colors of the branches indicate the population source for the viral ancestors over time. 

# [Between-population transmission](http://localhost:4000/UofISARSCoV2_UofI-2020-2021-AcademicYear?d=tree&f_population=Community,University&legend=open)
The places on the tree where the branch color changes from one population to another represent viral transmission between those populations. Most transmissions occurred within populations rather than between populations, indicating minimal viral spread between the Community and University throughout the academic year.

# [New introductions](http://localhost:4000/UofISARSCoV2_UofI-2020-2021-AcademicYear?d=tree&c=introduction_usher&f_population=Community,University&legend=closed)
New introductions of viruses from outside regions can be the starting point for viral spread within a population. We identified about 200 introductions into our study populations, but only 12 of those introductions led to large numbers of infections (i.e., led to > 10 infected individuals in our dataset). Here the descendants of these 12 introductions are highlighted in color. 

We found more introductions in the Fall than Spring semester, and the Fall introductions tended to leave more descendants over a longer period of time. This indicates more viral spread in the Fall not only from outside our study populations, but also within our study populations. 

# [Outbreaks](http://localhost:4000/UofISARSCoV2_UofI-2020-2021-AcademicYear?d=tree&c=outbreak&f_population=Community,University&legend=closed)
Outbreaks occur when large numbers of individuals become infected simultaneously, often due to super spreader events like large social gatherings. Genome sequence data allows us to identify outbreaks, which are characterized by large numbers of samples that have identical genome sequences. We identified a total of 21 outbreaks that had ten or more identical genome sequences, and these outbreaks are colored here. 

Outbreaks tended to be dominated by either University or Community samples (not both), and University-dominated outbreaks tended to be larger but last shorter lengths of time than Community-dominated outbreaks. 

# [Explore on your own!](http://localhost:4000/UofISARSCoV2_UofI-2020-2021-AcademicYear?d=tree&legend=open)
Click on "explore the data yourself" to explore the tree. You can click on the dots to learn more about each sample, and you can highlight samples by age, sex, variant, and other categories. You can also zoom into certain parts of the tree by clicking on the branches.


